### PR TITLE
Add api features to console logs for extra results information

### PR DESCRIPTION
**Pull request by jwu910**
Added js api features to the console log

Edit: Fixes https://github.com/5-gwoap/slng-node/issues/19

### CODE REVIEW

**Review by psyduc — APPROVED** [review]
lgtm

File: index.js [hunk 1]
```diff
@@ -34,7 +41,11 @@ program
         }
         trimRes.forEach((result) => {
           if (typeof (result.definition) !== undefined) {
-            console.log(chalk.bold.cyan('=======================\n') + result.definition);
+            console.log('Word: ' + chalk.magenta(result.word));
```
Comment (rainrivas) [inline index.js]: Magenta doesn't work on my powershell @jwu910

File: index.js [hunk 2]
```diff
@@ -24,6 +28,9 @@ program
       if (err) throw new Error(err);
       var trimRes;
       var results = JSON.parse(body).list;
+
+      console.log('='.repeat(consoleWidth()));
```
Comment (rainrivas) [inline index.js]: We removed cyan from our split, was this on purpose?

File: index.js (hunk 1 above)
Comment (rainrivas) [reply index.js]: image

**Review by rainrivas — CHANGES_REQUESTED** [review]
Magenta does not render on shells (ubuntu default, @psyduc to provide SS; and powershell, SS attached). 
The '=' split color was removed

File: index.js (hunk 1 above)
Comment (rainrivas) [reply index.js]: FYI highlighting does **not** help
image

Comment (psyduc) [reply index.js]: screenshot from 2018-01-21 22-43-27

**Review by rainrivas — APPROVED** [review]

**Comment by rainrivas** [comment]
@jwu910 if you cool with the color you can merge this to dev

**Pull request merged**